src/main/java/com/pretz/parkingmanager/mapper/ParkingStartDTOToParkingSessionConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.pretz.parkingmanager.mapper;
+
+import com.pretz.parkingmanager.domain.ParkingRate;
+import com.pretz.parkingmanager.domain.ParkingSession;
+import com.pretz.parkingmanager.dto.ParkingStartDTO;
+import org.modelmapper.Converter;
+import org.modelmapper.spi.MappingContext;
+
+public class ParkingStartDTOToParkingSessionConverter
+        implements Converter<ParkingStartDTO, ParkingSession> {
+
+    @Override
+    public ParkingSession convert(final MappingContext<ParkingStartDTO, ParkingSession> mappingContext) {
+
+        ParkingStartDTO source = mappingContext.getSource();
+        ParkingSession.ParkingSessionBuilder destination = ParkingSession.builder()
+                .vehicleId(source.getVehicleId());
+
+        if (source.getParkingRateId() == 1) {
+            destination.parkingRate(ParkingRate.REGULAR);
+        } else if (source.getParkingRateId() == 2) {
+            destination.parkingRate(ParkingRate.DISABLED);
+        } else
+            destination.parkingRate(ParkingRate.NONE);
```

Comment:
i co zrobimy z takim klientem?

Reply:
Nie będzie takiego klienta, bo to jest wcześniej wyłapywane w walidacji. :)

Zapewne tego całego NONE należałoby albo wywalić albo przemyśleć skąd właściwie ma być wpisywany. Bo wychodzi to trochę niespójnie